ci/requirements-3.3.run

```diff
@@ -14,3 +14,4 @@ lxml=3.2.1
 scipy
 beautiful-soup=4.2.1
 statsmodels
+Jinja2==2.8
```

Comment:
let one of these be the current version, iow no version specification (or >= if needed), (usually do this on 3.5)